server.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const knex = require('./db/config');
+
+
+const environment = process.env.NODE_ENV || 'development';
+const config = require('./knexfile')[environment];
+const database = require('knex')(config);
+
+const app = express();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.set('port', process.env.port || 5000);
+app.listen(app.get('port'));
+
+app.use('/', express.static(`${__dirname}/client/build`));
+
+app.get('/api/v1/test', (req, res) => {
+  return res.status(200).json({ status: 'success' });
```

Comment:
I don't think we need the `return` or `status(200)` (pretty sure express defaults to 200 status), but they're not hurting anything.